src/diffusers/pipelines/alt_diffusion/pipeline_alt_diffusion.py

```diff
@@ -303,7 +303,7 @@ def encode_prompt(
             self._lora_scale = lora_scale
 
             # dynamically adjust the LoRA scale
-            adjust_lora_scale_text_encoder(self.text_encoder, lora_scale)
+            adjust_lora_scale_text_encoder(self.text_encoder, lora_scale, self.use_peft_backend)
```

Comment:
Excellent use of reusing subclass variable (`use_peft_backend`) here!

Reply:
Thanks !